### DISCUSSION TITLE
How to Prevent content displayed above the safe area when scrolled in iOS app build using capacitor?

### DISCUSSION THREAD

**Discussion by neroshanbme36** [ionic-team/capacitor#3234] [Q&A]
I am building a mobile app from my angular project using capacitor without ionic. Since the app goes to full screen in iPhone the app overlaps the status barto fix that I have configured in capacitor.config.json as "ios": { "contentInset": "automatic" }

After adding the contentInset: automatic my layout is properly laid. But when i scroll up the scrolled content is displayed in the status bar since iphone have transparent status bar. Is there any way to hide the scrolled content in status bar while scrolling?

![image](https://user-images.githubusercontent.com/48980638/87121413-508a0580-c2a0-11ea-8025-2c7eb8405f97.png)

**Comment by luismiguelbravo** [comment]
add margin top or remove iphone status bar
